### ISSUE TITLE
The install script should install in local when the user is not root

### ISSUE THREAD

**Issue by simonbcn** [nana-4/materia-theme#280]
If I execute `install.sh` script like normal user, it should install the themes in `$HOME/.themes`.
The change is easy:
```
ROOT_UID=0
DEST_DIR=

# Destination directory
if [ "$UID" -eq "$ROOT_UID" ]; then
  DEST_DIR="/usr/share/themes"
else
  DEST_DIR="$HOME/.themes"
fi
```

**Comment by nana-4** [comment]
Hi, thanks for your suggestion.

For some reasons I basically want to force users to install the themes in `/usr/share/themes`. Because:

1. Some DEs can read themes from `$HOME/.themes`, but cannot read from `$HOME/.local/share/themes`. Some other DEs can read themes from `$HOME/.local/share/themes`, but cannot read from `$HOME/.themes`.
  `/usr/share/themes` is the only location where all DEs can read themes.

2. Forcing the location (by default) makes it easier to write the docs.

3. Users can specify the destination directory with `./install.sh -d DEST_DIR`.

In short, `/usr/share/themes` is the most trouble-free location, but `$HOME/.themes` sometimes gets into trouble.

So I would like to close this as WONTFIX, but if you have a good idea please let me know.
